src/commonMain/kotlin/exchange.dydx.abacus/processor/configs/FeeTierProcessor.kt

```diff
@@ -30,10 +30,10 @@ internal class FeeTierProcessor(parser: ParserProtocol) : BaseProcessor(parser)
         payload: IMap<String, Any>
     ): IMap<String, Any>? {
         val received = transform(existing, payload, feeTierKeyMap)
-        var absoluteVolumeRequirement = parser.asDecimal(payload["absoluteVolumeRequirement"])
+        val absoluteVolumeRequirement = parser.asDecimal(payload["absoluteVolumeRequirement"])
         if (absoluteVolumeRequirement != null) {
-            received["volume"] = absoluteVolumeRequirement
             received["symbol"] = "≥"
+            received["volume"] = absoluteVolumeRequirement / 1000000
```

Comment:
Should we add a constant variable for `1_000_000` and use that for division in all of these cases. Would be more clear for people wondering why we are dividing each of these items by `1_000_000`

Reply:
yes, moved it to util.constants